command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
@mikebrow sure, but no web spec exists so the only current mechanism to comply is via cli. This is just a fact. We can add new ways to comply later but today there is just one and I don't see how we write a compliance check without requiring compliance with it, at least for purposes of checking compliance. How about wording like this: "In order to be certified as compliant with the spec a certification tool MAY require that a runtime implement an approved API specification, so that the testing tool can invoke commands using it and validate output. The current list of approved API specifications includes only the CLI Spec" (or similar)